### ISSUE TITLE
Currency format dynamic fractional digits

### ISSUE THREAD

**Issue by gordoneliel** [elixir-cldr/cldr_numbers#36]
I'm using ex_money and formatting money in phoenix views and its fantastic. 

The default formatting works great, except what I want is for the formatted money to not include fractional digits if there is none / is zero. Eg. *$20.00* to be *$20*, but *$20.01* should remain the same.

Is there an option that does this? I'm curious how others go about this.

**Comment by kipcole9** [comment]
@gordoneliel there's no option to do what you're after automatically. However I have just published [ex_money version 5.14.0](https://hex.pm/packages/5.14.0) that includes a `Money.integer?/1` function. Which means in your code you can do something like:
```elixir
if Money.integer?(money) do
  Money.to_string money, fractional_digits: 0
else
  Money.to_string(money)
end
```
Which produces:
```elixir
iex> Fraction.to_string Money.new(:USD, 100)
{:ok, "$100"}
iex> Fraction.to_string Money.new(:USD, "100.10")
{:ok, "$100.10"}
```

**Comment by gordoneliel** [comment]
That works! I'll give it a shot, thanks for the help.
